lib/makeboarddeb.sh

```diff
@@ -179,7 +179,7 @@ create_board_package()
 	fi
 
 	EOF
-
+	##FIXME this is probably tech debt
```

Comment:
Yes. This is obsolete. Mesa went further and we ship focal / hirsute builds anyway.